reana/cli.py

```diff
@@ -1154,9 +1168,13 @@ def run_example(component, workflow_engine, output, sleep):  # noqa: D301
                         inputdir, workflow_name)
                     run_command(cmd, component)
             # run workflow
+            input_parameters = ' '.join(
+                ['-p ' + parameter for parameter in parameters])
+            operational_options = ' '.join(
+                ['-p ' + option for option in options])
```

Comment:
Note the typo here, should be `-o` instead of `-p` for options, otherwise LGTM

Reply:
Also `-o` flag is already taken - https://github.com/reanahub/reana/blob/6d921eebb301531e67aed684eb35bceb350973b4/reana/cli.py#L1108